diff --git a/package/lean/autocore/Makefile b/package/lean/autocore/Makefile
index 22b986d58c..945f9a2784 100644
--- a/package/lean/autocore/Makefile
+++ b/package/lean/autocore/Makefile
@@ -17,7 +17,7 @@ include $(INCLUDE_DIR)/package.mk
 define Package/autocore
   TITLE:=x86/x64 auto core loadbalance script.
   MAINTAINER:=Lean
-  DEPENDS:=@TARGET_x86 +bc +lm-sensors +ethtool
+  DEPENDS:=+bc +lm-sensors +ethtool
 endef
 
 define Package/autocore/description
diff --git a/package/lean/autocore/files/autocore b/package/lean/autocore/files/autocore
index 436e92940a..16262fb1c6 100755
--- a/package/lean/autocore/files/autocore
+++ b/package/lean/autocore/files/autocore
@@ -33,7 +33,7 @@ start()
 	g=${a}${b}${c}${d}${e}${f}
 
 	mkdir -p /tmp/sysinfo
-	echo $g > /tmp/sysinfo/model
+	#echo $g > /tmp/sysinfo/model
 
 	a=$(ip address | grep ^[0-9] | awk -F: '{print $2}' | sed "s/ //g" | grep '^[e]' | grep -v "@" | grep -v "\.")
 	b=$(echo "$a" | wc -l)
@@ -44,7 +44,7 @@ start()
 		ethtool -K $c tx-checksum-ip-generic on >/dev/null 2>&1 || (
 		ethtool -K $c tx-checksum-ipv4 on >/dev/null 2>&1
 		ethtool -K $c tx-checksum-ipv6 on >/dev/null 2>&1)
-		ethtool -K $c tx-scatter-gather on >/dev/null 2>&1
+		#ethtool -K $c tx-scatter-gather on >/dev/null 2>&1
 		ethtool -K $c gso on >/dev/null 2>&1
 		ethtool -K $c tso on >/dev/null 2>&1
 		ethtool -K $c ufo on >/dev/null 2>&1
diff --git a/package/lean/autocore/files/cpuinfo b/package/lean/autocore/files/cpuinfo
index 8c0527f84f..6adabeafc7 100755
--- a/package/lean/autocore/files/cpuinfo
+++ b/package/lean/autocore/files/cpuinfo
@@ -1,5 +1,5 @@
 #!/bin/sh
 
-MHz=`grep 'MHz' /proc/cpuinfo | cut -c11- |sed -n '1p'`
-TEMP=`sensors 2>/dev/null | grep 'Core 0' | cut -c12-`
+MHz=`echo "$(cat /sys/devices/system/cpu/cpu[04]/cpufreq/cpuinfo_cur_freq)/1000" | bc`
+TEMP=`echo "scale=2; $(cat /sys/class/thermal/thermal_zone0/temp)/1000" | bc`°
 echo "$MHz MHz $TEMP "
diff --git a/package/lean/autocore/files/rpcd_10_system.js b/package/lean/autocore/files/rpcd_10_system.js
index 73638a0b6b..dda369ba5c 100644
--- a/package/lean/autocore/files/rpcd_10_system.js
+++ b/package/lean/autocore/files/rpcd_10_system.js
@@ -76,7 +76,7 @@ return L.Class.extend({
 			_('Hostname'),         boardinfo.hostname,
 			_('Model'),            boardinfo.model + cpubench.cpubench,
 			_('CPU Info'),         cpuinfo.cpuinfo,
-			_('Firmware Version'), (L.isObject(boardinfo.release) ? boardinfo.release.description : '')/* + ' / ' : '') + (luciversion || '')*/,
+			_('Firmware Version'), (L.isObject(boardinfo.release) ? boardinfo.release.description : '') + ' built by DYC'/* + ' / ' : '') + (luciversion || '')*/,
 			_('Kernel Version'),   boardinfo.kernel,
 			_('Local Time'),       datestr,
 			_('Uptime'),           systeminfo.uptime ? '%t'.format(systeminfo.uptime) : null,
