diff --git a/package/lean/autocore/Makefile b/package/lean/autocore/Makefile
index b10064ba..2b702480 100644
--- a/package/lean/autocore/Makefile
+++ b/package/lean/autocore/Makefile
@@ -25,7 +25,7 @@ endef
 define Package/autocore-x86
   TITLE:=x86/x64 auto core loadbalance script.
   MAINTAINER:=Lean
-  DEPENDS:=@TARGET_x86 +bc +lm-sensors +ethtool
+  DEPENDS:=+bc +lm-sensors +ethtool
   VARIANT:=x86
 endef
 
diff --git a/package/lean/autocore/files/x86/autocore b/package/lean/autocore/files/x86/autocore
index 614e2176..5ca9d6cb 100755
--- a/package/lean/autocore/files/x86/autocore
+++ b/package/lean/autocore/files/x86/autocore
@@ -1,5 +1,4 @@
-#!/bin/sh /etc/rc.common
-# Copyright (C) 2017 lean <coolsnowwolf@gmail.com>
+
 
 START=99
 
@@ -31,7 +30,7 @@ start()
 	g=${a}${b}${c}${d}${e}${f}
 
 	mkdir -p /tmp/sysinfo
-	echo $g > /tmp/sysinfo/model
+	#echo $g > /tmp/sysinfo/model
 
 	a=$(ip address | grep ^[0-9] | awk -F: '{print $2}' | sed "s/ //g" | grep '^[e]' | grep -v "@" | grep -v "\.")
 	b=$(echo "$a" | wc -l)
@@ -42,7 +41,7 @@ start()
 		ethtool -K $c tx-checksum-ip-generic on >/dev/null 2>&1 || (
 		ethtool -K $c tx-checksum-ipv4 on >/dev/null 2>&1
 		ethtool -K $c tx-checksum-ipv6 on >/dev/null 2>&1)
-		ethtool -K $c tx-scatter-gather on >/dev/null 2>&1
+		#ethtool -K $c tx-scatter-gather on >/dev/null 2>&1
 		ethtool -K $c gso on >/dev/null 2>&1
 		ethtool -K $c tso on >/dev/null 2>&1
 		ethtool -K $c ufo on >/dev/null 2>&1
diff --git a/package/lean/autocore/files/x86/index.htm b/package/lean/autocore/files/x86/index.htm
index b0b56033..cbf45bc2 100644
--- a/package/lean/autocore/files/x86/index.htm
+++ b/package/lean/autocore/files/x86/index.htm
@@ -718,6 +718,7 @@
 		<tr><td width="33%"><%:Firmware Version%></td><td>
 			<%=pcdata(ver.distname)%> <%=pcdata(ver.distversion)%> <!--/
 			<%=pcdata(ver.luciname)%> (<%=pcdata(ver.luciversion)%>) !-->
+			built by DYC
 		</td></tr>
 		<tr><td width="33%"><%:Kernel Version%></td><td><%=unameinfo.release or "?"%></td></tr>
 		<tr><td width="33%"><%:Local Time%></td><td id="localtime">-</td></tr>
diff --git a/package/lean/autocore/files/x86/sbin/cpuinfo b/package/lean/autocore/files/x86/sbin/cpuinfo
index 8c0527f8..575c620f 100755
--- a/package/lean/autocore/files/x86/sbin/cpuinfo
+++ b/package/lean/autocore/files/x86/sbin/cpuinfo
@@ -1,5 +1,5 @@
 #!/bin/sh
 
-MHz=`grep 'MHz' /proc/cpuinfo | cut -c11- |sed -n '1p'`
-TEMP=`sensors 2>/dev/null | grep 'Core 0' | cut -c12-`
+MHz=`echo "$(cat /sys/devices/system/cpu/cpu[04]/cpufreq/cpuinfo_cur_freq)/1000" | bc`
+TEMP=`echo "scale=2; $(cat /sys/class/thermal/thermal_zone0/temp)/1000" | bc`
 echo "$MHz MHz $TEMP "
