diff --git a/package/lean/autocore/Makefile b/package/lean/autocore/Makefile
index 5a3ea113..1f52537b 100644
--- a/package/lean/autocore/Makefile
+++ b/package/lean/autocore/Makefile
@@ -17,7 +17,7 @@ include $(INCLUDE_DIR)/package.mk
 define Package/autocore
   TITLE:=x86/x64 auto core loadbalance script.
   MAINTAINER:=Lean
-  DEPENDS:=@TARGET_x86 +bc +lm-sensors +ethtool
+  DEPENDS:=+bc +lm-sensors +ethtool
 endef
 
 define Package/autocore/description
diff --git a/package/lean/autocore/files/autocore b/package/lean/autocore/files/autocore
index 55cd519f..92873a1c 100755
--- a/package/lean/autocore/files/autocore
+++ b/package/lean/autocore/files/autocore
@@ -31,6 +31,7 @@ start()
 	e=$(cat /proc/cpuinfo | grep 'processor' | wc -l)
 	f=$(echo -n ' Thread ')
 	g=${a}${b}${c}${d}${e}${f}
+	g='FriendlyElec NanoPi R2S'${b}${e}${d}
 
 	mkdir -p /tmp/sysinfo
 	echo $g > /tmp/sysinfo/model
@@ -50,5 +51,5 @@ start()
 		ethtool -K $c ufo on >/dev/null 2>&1
 	done
 
-	[ -f /etc/index.htm ] && mv /etc/index.htm /usr/lib/lua/luci/view/admin_status/index.htm
+	#[ -f /etc/index.htm ] && mv /etc/index.htm /usr/lib/lua/luci/view/admin_status/index.htm
 }
