diff --git a/package/lean/luci-app-cpufreq/root/etc/config/cpufreq b/package/lean/luci-app-cpufreq/root/etc/config/cpufreq
index 0443dc77..214d7cb8 100644
--- a/package/lean/luci-app-cpufreq/root/etc/config/cpufreq
+++ b/package/lean/luci-app-cpufreq/root/etc/config/cpufreq
@@ -1,6 +1,6 @@
 
 config settings 'cpufreq'
-	option maxfreq '716000'
+	option maxfreq '1296000'
 	option upthreshold '50'
 	option factor '10'
 	option minifreq '300000'
diff --git a/package/lean/luci-app-cpufreq/root/etc/init.d/cpufreq b/package/lean/luci-app-cpufreq/root/etc/init.d/cpufreq
index df42f3c7..c81e237d 100755
--- a/package/lean/luci-app-cpufreq/root/etc/init.d/cpufreq
+++ b/package/lean/luci-app-cpufreq/root/etc/init.d/cpufreq
@@ -13,7 +13,7 @@ start()
 	config_load cpufreq
 	local governor=$(uci_get_by_type settings governor ondemand)
 	local minifreq=$(uci_get_by_type settings minifreq 48000)
-	local maxfreq=$(uci_get_by_type settings maxfreq 716000)
+	local maxfreq=$(uci_get_by_type settings maxfreq 1296000)
 	local upthreshold=$(uci_get_by_type settings upthreshold 50)
 	local factor=$(uci_get_by_type settings factor 10)
 	
